flixel/sound/FlxSound.hx

```diff
@@ -93,18 +93,27 @@ class FlxSound extends FlxBasic
 	 * Set volume to a value between 0 and 1 to change how this sound is.
 	 */
 	public var volume(get, set):Float;
+
 	#if FLX_PITCH
 	/**
 	 * Set pitch, which also alters the playback speed. Default is 1.
 	 */
 	public var pitch(get, set):Float;
 	#end
+
 	/**
 	 * The position in runtime of the music playback in milliseconds.
 	 * If set while paused, changes only come into effect after a `resume()` call.
 	 */
 	public var time(get, set):Float;
 
+	#if (native && !openfl_legacy)
```

Comment:
Rather than using `#if (blah blah blah)` we need to define a FLX_SOUND_GAIN for devs to use if they are doing multitargeting

Follow [what FLX_PITCH did](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/system/macros/FlxDefines.hx#L151-L152)

Reply:
Ok